src/rxjsPipeableOperatorsOnlyRule.ts

```diff
@@ -206,7 +206,12 @@ function findImportedRxjsOperators(sourceFile: ts.SourceFile): Set<string> {
  */
 function createImportReplacements(operatorsToAdd: Set<string>, startIndex: number): Lint.Replacement[] {
   return [...Array.from(operatorsToAdd.values())].map(operator =>
-    Lint.Replacement.appendText(startIndex, `\nimport {${operator}} from 'rxjs/operators/${operator}';\n`)
+    NGRX_OPERATORS.has(operator)
+      ? Lint.Replacement.appendText(startIndex, `\nimport {${operator}} from '@ngrx/store';\n`)
```

Comment:
Just found that this needs to be more dynamic based on import type as `ofType` needs to come from `@ngrx/effects`